src/app/static/src/app/store/surveyAndProgram/mutations.ts

```diff
@@ -37,5 +38,9 @@ export const mutations: MutationTree<SurveyAndProgramDataState> & SurveyAndProgr
 
     ANCError(state: SurveyAndProgramDataState, action: PayloadWithType<string>) {
         state.ancError = action.payload;
+    },
+
+    Ready(state: SurveyAndProgramDataState) {
```

Comment:
Since this mutation is going to be in (almost) every step, could we have a base mutation object with a Ready(state: InputState) function on it, which the other mutation objects 'inherit' from via {...baseMutations}?